pkg/client/client_test.go

```diff
@@ -82,6 +82,25 @@ var _ = Describe("ConmonClient", func() {
 				}, time.Second*20).Should(BeNil())
 			})
 
+			It(testName("should execute cleanup command when container exits", terminal), func() {
+				tr = newTestRunner()
+				tr.createRuntimeConfig(terminal)
+				sut = tr.configGivenEnv()
+				tr.createContainerWithConfig(sut, &client.CreateContainerConfig{
+					ID:           tr.ctrID,
+					BundlePath:   tr.tmpDir,
+					Terminal:     terminal,
+					ExitPaths:    []string{tr.exitPath()},
+					OOMExitPaths: []string{tr.oomExitPath()},
+					LogDrivers: []client.LogDriver{{
+						Type: client.LogDriverTypeContainerRuntimeInterface,
+						Path: tr.logPath(),
+					}},
+					CleanupCmd: "echo cleanup command executed",
```

Comment:
can we have this be something that `touch`es a file to the temp dir, then we can verify that file is created once the container stops?